packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -1019,6 +1021,7 @@ export class AppMainContainer extends Component<
           accept=".json"
           style={{ display: 'none' }}
           onChange={this.handleImportLayoutFiles}
+          data-testid="btn-import-layout"
```

Comment:
Yea, I'm following the guide here: https://testing-library.com/docs/user-event/utility/#upload
Since we trigger that inputElement button from a click on another button, I don't know how to simulate that - clicking the "Import Layout" button doesn't trigger anything (that I can tell), and trying to .upload() on it just throws an error. So just added data-testid to the input element.

I do need to clean up testing a bit though.

Reply:
Makes sense. I was mostly commenting on the testid itself should be `input-import-layout` probably